docs/RELEASE.md

```diff
@@ -0,0 +1,136 @@
+# Release
+
+The outputs of a release are the `polkadot` and `polkadot-parachain` nodes, runtimes for the Westend & Rococo networks, including their system parachains, and new crate versions published to `crates.io`.
+
+# Setup
+
+We have two branches: `master` and `release`. `master` is the main development branch where normal merge requests are opened. Developers need to mostly only care about this branch.  
+The `release` branch contains a version of the code that is ready to be released. Its contents are always audited. Merging to it is restricted to [Backports](#backports).
+
+# Versioning
+
+We are releasing multiple different things from this repository in one release, but 
+we don't want to use the same version for everything. Thus, in the following we explain
+the versioning story for the crates, node and Westend & Rococo. To easily refer to a release, it shall be named by its date in the form `stableYYMMDD`.
+
+## Crate
+
+We try to follow SemVer<sup>1</sup> as best as possible for versioning our crates. SemVer requires a piece of software to first declare a public API. The public API of the Polkadot SDK is hereby declared as the sum of all crates' public APIs, with the exception of the following internal crates:
+
+- @bkchr the list of node internals please
+
+The public API of our library crates is declared as all public items that are not inside a `__private` module.
```

Comment:
...And also not marked explicitly as "unstable" either in the first line of documentation or within the API entity's name or path.